### PR TITLE
Moves corporate items and datums into a corporate 'content package'.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
This is something of a proof of concept and example for discussion. Essentially just moving non-core content into a folder, `content_packages`, where all the files are unticked, and there is a root `_foo.dm` which contains the #include statements needed to compile the content. The idea then being that a map that wants the corporate content would `#include "..\..\code\content_packages\_corporate.dm` and get it all, while downstreams ~~like Scav~~ can just shift-delete the entire folder to remove it from their codebase.

Depends on #47.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
This is finally ready for review.

File: maps/~mapsystem/maps.dm [hunk 1]
```diff
@@ -93,7 +94,7 @@ var/const/MAP_HAS_RANK = 2		//Rank system, also togglable
 	var/list/lobby_tracks = list()                  // The list of lobby tracks to pick() from. If left unset will randomly select among all available /music_track subtypes.
 	var/welcome_sound = 'sound/AI/welcome.ogg'		// Sound played on roundstart
 
-	var/default_law_type = /datum/ai_laws/nanotrasen  // The default lawset use by synth units, if not overriden by their laws var.
+	var/default_law_type = /datum/ai_laws/asimov  // The default lawset use by synth units, if not overriden by their laws var.
```
Comment (comma) [inline maps/~mapsystem/maps.dm:97]: jesus isn't there a generic 4 laws to use rather than this monstrosity?

Comment (MistakeNot4892) [reply maps/~mapsystem/maps.dm:97]: Per convo on Discord - personally I like Asimov. I feel like the generic Bay lawsets are all very boring and bland - Asimov allows for more inround stupidity.

Bearcat doesn't use it as the default anyway.

File: maps/away_sites_testing/away_sites_testing.dm [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
-	#include "..\..\code\content_packages\misc\mundane.dm"
-	#include "..\..\code\content_packages\misc\spacemen.dm"
-	#include "..\..\code\content_packages\corporate\_corporate.dm"
+	#ifndef CONTENT_PACKAGE_MUNDANE
+		#include "..\..\code\content_packages\misc\mundane.dm"
+	#endif
+	#ifndef CONTENT_PACKAGE_SPACEMEN
+		#include "..\..\code\content_packages\misc\spacemen.dm"
+	#endif
+	#ifndef CONTENT_PACKAGE_CORPORATE
```
Comment (PsiOmegaDelta) [inline maps/away_sites_testing/away_sites_testing.dm]: Couldn't these define checks be in the content files themselves?
I.e. in `in _corporate.dme`
```
#ifndef CONTENT_PACKAGE_CORPORATE
#define CONTENT_PACKAGE_CORPORATE
rest of code in _corporate.dme
#endif
```

That way they can be included without concern elsewhere

Comment (MistakeNot4892) [reply maps/away_sites_testing/away_sites_testing.dm]: Good point.

**Review by afterthought2 — APPROVED** [review]

**Pull request merged**